src/Paysera/WalletApi/Entity/User/PoliticallyExposedPerson.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+class Paysera_WalletApi_Entity_User_PoliticallyExposedPerson
+{
+    /**
+     * @var string
+     */
+    protected $name;
+
+    /**
+     * @var string
+     */
+    protected $relation;
+
+    /**
+     * @var array
+     */
+    protected $positions;
+
+    /**
+     * Creates object, used for fluent interface
```

Comment:
Do we need this comment?

Reply:
Yeah I would leave it. We have it in every same method so I would leave it because people might think about why we have this without comment